index.bs

```diff
@@ -11876,33 +11892,24 @@ the [=primary interface=]
 of the platform object.
 
 
-<h4 id="indexed-and-named-properties">Indexed and named properties</h4>
+<h3 id="es-legacy-platform-objects" oldids="indexed-and-named-properties">Legacy platform objects</h3>
 
-If a [=platform object=] implements an [=interface=] that
-[=support indexed properties|supports indexed=] or
-[=support named properties|named properties=],
-the object will appear to have additional properties that correspond to the
-object’s indexed and named properties.  These properties are not “real” own
+[=Legacy platform objects=] will appear to have additional properties that correspond to the
+[=indexed properties|indexed=] and [=named properties=].  These properties are not “real” own
 properties on the object, but are made to look like they are by being exposed
 by the \[[GetOwnProperty]] internal method.
 
-However, when the [{{Global}}] or
-[{{PrimaryGlobal}}]
-[=extended attribute=] has been used,
-named properties are not exposed on the object but on another object
-in the prototype chain, the [=named properties object=].
-
 It is permissible for an object to implement multiple interfaces that support indexed properties.
```

Comment:
Maybe as a follow-up we should disallow this. @bzbarsky, do you know of any existing interfaces that implement one or more interfaces that support indexed properties? (Or for that matter named properties.)